programs/relayer/src/modules/cosmos_to_eth.rs

```diff
@@ -66,14 +66,17 @@ pub enum SP1Config {
     /// The network prover.
     Network {
         /// The optional private key for the network prover.
+        /// Only used when `prover_type` is "network".
```

Comment:
This is not needed in rustdocs as prover type has to be network for this to be accessible
```suggestion
```